msyncd/SyncDBusInterface.h

```diff
@@ -166,6 +166,16 @@ class SyncDBusInterface : public QObject
      */
     bool isConnectivityAvailable(int connectivityType);
 
+    /*! \brief Notifies always-up-to-date state for an account
+     *
+     * This signal is sent when the always-up-to-date state of a particular
+     * account ID changes or a client queries for the state via 'isAlwaysOn'
+     *
+     * \param aAccountId The account IDs that changed state/was queried the state
+     * \param aState The current always-up-to-date state(on/off).
+     */
+    void alwaysOnState(uint aAccountId, bool aState);
```

Comment:
Should this be per-profile?  A single account might have separate sync profiles, some of which are controlled externally, and some which are controlled by buteo...